components/Ink/src/MDCInkView.m

```diff
@@ -193,7 +205,17 @@ - (CGFloat)maxRippleRadius {
 }
 
 - (void)setMaxRippleRadius:(CGFloat)radius {
-  if (self.inkLayer.maxRippleRadius != radius) {
+  _maxRippleRadius = radius;
```

Comment:
aren't we losing the original maxRippleRadius here if we set it to 0 when bounded? so when it is set to unbounded we will get 0 and not the original value

Reply:
I'm not sure I understand. The ivar was not previously being used and now it's only used to "restore" the assigned value when switching to `.unbounded` ink.

Reply:
Just tested my hypothesis, this seems fine, please ignore.